game.py

```diff
@@ -14,13 +14,37 @@
 
 
 def snowman(snowman_word):
-    """Complete the snowman function
-    replace "pass" below with your own code
-    It should print 'Congratulations, you win!'
-    If the player wins and, 
-    'Sorry, you lose! The word was {snowman_word}' if the player loses
-    """
-    pass
+    # The word is provided as a parameter (tests pass it in)
```

Comment:
From the [PEP 8 Style Guide](https://peps.python.org/pep-0008/#inline-comments): 

*"Use inline comments sparingly.*

*An inline comment is a comment on the same line as a statement. Inline comments should be separated by at least two spaces from the statement. They should start with a # and a single space.*

*Inline comments are unnecessary and in fact distracting if they state the obvious..."*

We only want to use comments when absolutely necessary. Unnecessary comments will start to clutter our code base. You'll want to remove most, if not all, the comments you have here.